### ISSUE TITLE
checkpoints didn't perform as what paper shown

### ISSUE THREAD

**Issue by languandong** [zhjohnchan/M3AE#19]
Excellent work! 
As for VQA-RAD dataset of the VQA task, I download the checkpoint you provided  and run the test script.
The effect was shown below.
![image](https://github.com/zhjohnchan/M3AE/assets/44127403/f693e0b1-1b12-435d-9d27-bc56495111f9)

But paper's 
![image](https://github.com/zhjohnchan/M3AE/assets/44127403/9c585d95-8e2b-4961-b432-d3c867e2aaa3)

**Comment by dongzizhu** [comment]
Hi,
I'm also reimplementing their experiments. Could you share where did you get their dataset? I did find the VQA-RAD dataset according to the original RAD paper, but the train-val-test split is not clear. BTW I'm getting a much worse performance than yours.

Thx!

**Comment by languandong** [comment]
> Hi, I'm also reimplementing their experiments. Could you share where did you get their dataset? I did find the VQA-RAD dataset according to the original RAD paper, but the train-val-test split is not clear. BTW I'm getting a much worse performance than yours.
> 
> Thx!
I get the train-val-test  from a previous work MVEF  https://github.com/aioz-ai/MICCAI19-MedVQA.
It seems that all the work later was divided according to this

**Comment by languandong** [comment]
> Hi, I'm also reimplementing their experiments. Could you share where did you get their dataset? I did find the VQA-RAD dataset according to the original RAD paper, but the train-val-test split is not clear. BTW I'm getting a much worse performance than yours.
> 
> Thx!

I get the train-val-test  from a previous work MVEF  https://github.com/aioz-ai/MICCAI19-MedVQA.
It seems that all the work later was divided according to this

**Comment by dongzizhu** [comment]
> I get the train-val-test from a previous work MVEF https://github.com/aioz-ai/MICCAI19-MedVQA. It seems that all the work later was divided according to this

Yes yes, I'm also using the dataset here. Thx! Also, I think your results 0.659-0.849-0.7738 are close enough to their reported results :). Good luck!

**Comment by VcRenOne** [comment]
> Excellent work! As for VQA-RAD dataset of the VQA task, I download the checkpoint you provided and run the test script. The effect was shown below. ![image](https://private-user-images.githubusercontent.com/44127403/290054466-f693e0b1-1b12-435d-9d27-bc56495111f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMxNDMwMTMsIm5iZiI6MTcwMzE0MjcxMywicGF0aCI6Ii80NDEyNzQwMy8yOTAwNTQ0NjYtZjY5M2UwYjEtMWIxMi00MzVkLTlkMjctYmM1NjQ5NTExMWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIxVDA3MTE1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmZDkwOTU3YTI1M2RmMmIzMWE2YTA4OTcwYzNkZmQ3MTAxMTYxN2JjNTk1Y2U3MjRkYzk4OWQyMzJmYzhjOTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.63r6BG20Rm_tq0E4A3hzq4WFD6hyB_AhW0W-1WFOAzI)
> 
> But paper's ![image](https://private-user-images.githubusercontent.com/44127403/290054936-9c585d95-8e2b-4961-b432-d3c867e2aaa3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMxNDMwMTMsIm5iZiI6MTcwMzE0MjcxMywicGF0aCI6Ii80NDEyNzQwMy8yOTAwNTQ5MzYtOWM1ODVkOTUtOGUyYi00OTYxLWI0MzItZDNjODY3ZTJhYWEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIxVDA3MTE1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhNzc5MGUyNTEwNzc5ZWE4MTM1OWEwNTM2Y2UxMGQ2OTQ2MjAzMGJkNjczM2UxMWFkOGI4ZDFmZGMwYjZlZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.snrKsu62zDq4TODaaW9mwwPvchvq_ZywVRwNfrY3aKc)

Hi dalao, I'm reproducing this experiment on my own dataset, and it works, but without any val results or any test results. Would like to ask you how to reproduce the code, thank you very much!!
![微信截图_20231221151917](https://github.com/zhjohnchan/M3AE/assets/125938006/7ca5135b-178a-449a-80e4-bd833b6b0d5a)
It gets stuck in this spot (rank loop) every time
Also, I can't find any saved ckpt file

**Comment by Victory0622** [comment]
> > [我从之前的工作 MVEF https://github.com/aioz-ai/MICCAI19-MedVQA](https://github.com/aioz-ai/MICCAI19-MedVQA)获得了 train-val-test 。看来后来的工作都是按照这个来划分的
> 
> 是的，我一直在使用这里的数据集。谢谢！另外，我认为您的结果 0.659-0.849-0.7738 与他们报告的结果足够接近:)。祝你运好！

Hi, could you please share with me the split dataset including training, validation, and testing datasets? I have downloaded the previous work MVEF, but it only contains training and testing datasets. When I tried to divide it into training, validation, and testing datasets, I got worse results.

**Comment by Shmily17** [comment]
> > 嗨，我也在重新实现他们的实验。你能分享一下你从哪里得到他们的数据集吗？我确实根据原始 RAD 论文找到了 VQA-RAD 数据集，但 train-val-test 拆分尚不清楚。顺便说一句，我的表现比你差得多。
> > 感谢！
> 
> 我从以前的工作 MVEF https://github.com/aioz-ai/MICCAI19-MedVQA 中获得了 train-val-test。看来后来所有的工作都是按照这个划分的

Hello, I also encountered the same problem, the final accuracy rate is only 26%, please share the training set, validation set and test set, thank you from the bottom of my heart! Have a great day

**Comment by dongzizhu** [comment]
Hi @Shmily17 @Victory0622 

I think the reason you are getting a worse performance is because the label is messed up. I don't remember how I solved this, maybe I retrained the model or did some kind of label matching in the test process. But their code did work out. Good luck!

**Comment by shiyunchengAI** [comment]
> Excellent work! As for VQA-RAD dataset of the VQA task, I download the checkpoint you provided and run the test script. The effect was shown below. ![image](https://github.com/zhjohnchan/M3AE/assets/44127403/f693e0b1-1b12-435d-9d27-bc56495111f9)
> 
> But paper's ![image](https://github.com/zhjohnchan/M3AE/assets/44127403/9c585d95-8e2b-4961-b432-d3c867e2aaa3)


Excuse me, this is my first time using this code repository, and I found that the dataset link is not working.
I wonder if you would be kind enough to provide some help regarding the data.
I would be very grateful.
